setup.py

```diff
@@ -26,9 +26,9 @@
     packages=find_packages(),
     install_requires=INSTALL_REQUIRES,
     extras_require={
-        'aws': ["boto3>=1.9.66", "botocore>=1.12.66"],
-        "gcp": ["google-api-python-client>=1.7.8", "oauth2client>=4.1.3", "google-cloud-datastore>=1.7.3"],
-        "azure": ["azure-cosmosdb-table>=1.0.5", "bson>=0.5.8"]
+        'aws': ["boto3>=1.34.110", "botocore>=1.34.100"],
```

Comment:
See if we can upgrade to latest 
boto3 1.34.151
botocore 1.34.151
google-api-python-client 2.139.0